docs/painless/painless-api-reference/painless-api-reference-interval/packages.asciidoc

```diff
@@ -0,0 +1 @@
+
```

Comment:
I wonder if we shouldn't emit empty files. It isn't a big deal but I figure it'd be nice.

Reply:
Agreed, and I've thought of this issue too.  I would prefer to add this in a follow up PR as this one is already getting long.

Reply:
:+1:

Reply:
Decided to fit this one in and fix it.